docs/preview/yamlgettingstarted-triggers.md

```diff
@@ -32,8 +34,41 @@ trigger:
 
 Note, path filters are only supported for Git repositories in VSTS.
 
-## CI is opt-out
+### CI is opt-out
 
 Continuous integration builds can be turned off by specifying `trigger: none`
 
 Optionally, the triggers can be managed from the web definition editor, on the Triggers tab.
+
+## Pull request triggers (currently for GitHub repositories only)
+
+Pull request builds are on by default for all branches. Optionally an explicit list of branch filters can
+be specified in the YAML file.
+
+When a PR is updated, the YAML file from the target branch is used to evaluate the branch filters.
+
+And the workflow defined in the YAML file from the merge branch, is used to execute the build.
+
+### Simple PR trigger syntax
+
+A simple list of branches can be specified in the file, to control which branches trigger a PR build.
+
+For example, a simple list of inclusive branch filters may look like:
+
+```yaml
+trigger:
+- master
+- releases/*
+```
+
+### Full PR trigger syntax
+
+For more control, an alternative PR syntax is available:
+
+```yaml
+pr:
+  forks: true # whether to build PRs from forks
+  branches:
+    include: [string] # todo: examples
+    exclude: [string]
+```
```

Comment:
This needs to be much more extensive to be scale out to large projects.  Specifically:
- Optional vs. non-optional
- Included/excluded paths
- Included/excluded users
- Custom expressions (e.g. PR text contains "skip ci")

Reply:
Also:

- GitHub Context